### PR TITLE
fork conflict test

### PR DESCRIPTION
**Pull request by SeanChiu9418**
It is a fork conflict test

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -1,9 +1,4 @@
 TEST
-Merge 2
-Merge Test
-Merge Tutorial
-Merge 3
-add Test2
-add Teest3
+ForkBranch2
```
Comment (SeanChiuNKG) [inline README.txt]: This line should be remove

**Comment by SeanChiuNKG** [comment]
Please remore the line

File: README.txt (hunk 1 above)
Comment (SeanChiu9418) [reply README.txt]: Remove one but add one

**Comment by SeanChiu9418** [comment]
I remove one but add an important one

File: README.txt [hunk 2]
```diff
@@ -1,9 +1,4 @@
 TEST
-Merge 2
-Merge Test
-Merge Tutorial
-Merge 3
-add Test2
-add Teest3
+This line is important
```
Comment (SeanChiuNKG) [inline README.txt]: This line is not important also, please remove it

**Comment by SeanChiuNKG** [comment]
Please check my comment

File: README.txt (hunk 2 above)
Comment (SeanChiu9418) [reply README.txt]: OK ~ I will do it

Comment (SeanChiu9418) [reply README.txt]: Remove it

**Comment by SeanChiu9418** [comment]
I remove it please check

File: README.txt [hunk 3]
```diff
@@ -1,9 +1,4 @@
 TEST
-Merge 2
-Merge Test
-Merge Tutorial
-Merge 3
-add Test2
-add Teest3
+adfadada
```
Comment (SeanChiuNKG) [inline README.txt]: This should be a typo

**Review by SeanChiuNKG — COMMENTED** [review]
In Finish your review  window

File: README.txt (hunk 3 above)
Comment (SeanChiu9418) [reply README.txt]: really? please check it again

**Review by SeanChiuNKG — COMMENTED** [review]
In Finish your review #2

File: README.txt (hunk 3 above)
Comment (SeanChiuNKG) [reply README.txt]: It is a typo

Comment (SeanChiu9418) [reply README.txt]: OK got is

**Comment by SeanChiu9418** [comment]
I update please check it

**Review by SeanChiuNKG — APPROVED** [review]
It is OK

**Pull request closed without merging**